shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -0,0 +1,308 @@
+  rotation_degree_ = degree;
+  // Compute renderer transformation based on the angle of rotation.
+  double rad = (360 - rotation_degree_) * M_PI / 180;
+  auto geometry = flutter_tizen_window_->GetWindowGeometry();
+  int32_t width = geometry.width;
+  int32_t height = geometry.height;
+
+  double trans_x = 0.0, trans_y = 0.0;
+  if (rotation_degree_ == 90) {
+    trans_y = height;
+  } else if (rotation_degree_ == 180) {
+    trans_x = width;
+    trans_y = height;
+  } else if (rotation_degree_ == 270) {
+    trans_x = width;
+  }
+
+  // view settransform
+  flutter_trans_formation_ = {
+      cos(rad), -sin(rad), trans_x,  // x
+      sin(rad), cos(rad),  trans_y,  // y
+      0.0,      0.0,       1.0       // perspective
+  };
+
+  if (rotation_degree_ == 90 || rotation_degree_ == 270) {
+    std::swap(width, height);
+  }
+
+  flutter_tizen_window_->ResizeRenderTargetWithRotation(
+      {geometry.left, geometry.top, width, height}, rotation_degree_);
+
+  // Window position does not change on rotation regardless of its orientation.
+  SendWindowMetrics(geometry.left, geometry.top, width, height, 0.0);
+}
+
+void FlutterTizenView::OnPointerMove(double x,
+                                     double y,
+                                     size_t timestamp,
+                                     FlutterPointerDeviceKind device_kind,
+                                     int32_t device_id) {
+  if (pointer_state_) {
+    SendFlutterPointerEvent(kMove, x, y, 0, 0, timestamp, device_kind,
+                            device_id);
+  }
+}
+
+void FlutterTizenView::OnPointerDown(double x,
+                                     double y,
+                                     size_t timestamp,
+                                     FlutterPointerDeviceKind device_kind,
+                                     int32_t device_id) {
+  pointer_state_ = true;
+  SendFlutterPointerEvent(kDown, x, y, 0, 0, timestamp, device_kind, device_id);
+}
+
+void FlutterTizenView::OnPointerUp(double x,
+                                   double y,
+                                   size_t timestamp,
+                                   FlutterPointerDeviceKind device_kind,
+                                   int32_t device_id) {
+  pointer_state_ = false;
+  SendFlutterPointerEvent(kUp, x, y, 0, 0, timestamp, device_kind, device_id);
+}
+
+void FlutterTizenView::OnScroll(double x,
+                                double y,
+                                double delta_x,
+                                double delta_y,
+                                int scroll_offset_multiplier,
+                                size_t timestamp,
+                                FlutterPointerDeviceKind device_kind,
+                                int32_t device_id) {
+  SendFlutterPointerEvent(
+      pointer_state_ ? kMove : kHover, x, y, delta_x * scroll_offset_multiplier,
+      delta_y * scroll_offset_multiplier, timestamp, device_kind, device_id);
```

Comment:
Would it make more sense to apply `kScrollOffsetMultiplier` inside this method without accepting it as an argument?